### DISCUSSION TITLE
How top replace `$(DotNetVersion)-windows10.0.19041` to `$(DotNetVersion)-windows10.0.22621` in single project?

### DISCUSSION THREAD

**Discussion by xperiandri** [unoplatform/uno#16504] [Q&A] [answered]
I want to replace the Windows version from the default `windows10.0.19041` to `windows10.0.22621`.
But after I've done
```
    <TargetFrameworks>
      $(DotNetVersion)-android;
      $(DotNetVersion)-ios;
      $(DotNetVersion)-maccatalyst;
      $(DotNetVersion)-windows10.0.22621;
      $(DotNetVersion);
      $(DotNetVersion)-desktop;
      $(DotNetVersion)-browserwasm;
    </TargetFrameworks>
```

I see

![image](https://github.com/unoplatform/uno/assets/2365592/cae7feb3-4a93-4bb6-aaa7-a989d78adbbe)

**Comment by jeromelaban** [comment, marked as answer]
This looks like something is keeping the old target. Could you try closing the solution, delete the `Console.App.csproj.user` file and see if it's working again?

**Reply by xperiandri** [reply]
Indeed, that helped!

**Reply by xperiandri** [reply]
Builds now

**Reply by jeromelaban** [reply]
Thanks. Let's track here: https://github.com/unoplatform/uno/issues/16507

**Reply by agneszitte** [reply]
@xperiandri https://github.com/unoplatform/uno/issues/16507 is now closed. The related changes are now part of [5.2.121](https://www.nuget.org/packages/Uno.Sdk/5.2.121)

**Reply by xperiandri** [reply]
Great, thanks!
